src/index.js

```diff
@@ -1,6 +1,15 @@
 const fibonacci = (n) => {
-
+  var secuencia = [];
+  var a, b, resultado;
```

Comment:
Don't use `var`. Use `const` and `let`. You don't control scopes if you use `var`.